content/guide/netlify.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Netlify for Quire
+weight: 215
+type: essay
+---
+
+## Netlify 
+
+{{< q-figure id="netlify-logo" >}}
+
+> <br/> Netlify is a web developer platform that multiplies productivity. <br/> <br/> By unifying the elements of the modern decoupled web, from local development to advanced edge logic, Netlify enables a 10x faster path to much more performant, secure, and scalable websites and apps. Our bet on the JAMstack is quickly coming true. <br/> <br/>The web is rapidly changing away from monolithic to decoupled apps, and web developers are storming ahead with more power than ever. Netlify is built to cater to that movement, and in just a few years we’ve on-boarded more than half a million businesses and developers, and are building and serving millions of web projects daily around the globe.<br/> <br/>
+
+For Quire, Netlify can automatically generate a preview site to share with your collaborators every time you push changes to your project, and can also use it to host your final project when its ready to publish.
+
+## Steps to deploy to Netlify
+
+- [As stated in the previous page add your code to Github repositiory](/guide/github/)
+- Next sign up to Netlify here -> https://app.netlify.com/signup
+- Connect your Netlify account to your Github or Gitlab account
+
+
+### Netlify Build Configuration
+
+Once you accounts are connected you will asked to provide a `Production` directory or `Build` directory. Instead of doing this, which is fine, my recommendation is heading back to your repository and creating a `netlify.toml` file which will run commands from the root directory. These commands are set in the scripts block in the your `package.json` file.  In quire-starter-theme the path is `themes/quire-starter-theme/package.json`.
+Once you find your `package.json` file add `"build": "webpack --config webpack/webpack.config.prod.js && cd ../../ && hugo --minify --config config.yml,config/site.yml"` to `scripts` block like below:
+
+```json
```

Comment:
@naeluh could we add the script in to the `package.json` file ourselves, now, and just have it sitting there ready for Netlfiy to use rather than having someone have to add it manually? Maybe we namespace it so it's not accidentally used for something else, like `netlify-build`.

Reply:
Hi @geealbers yeah for sure, I was thinking about that. It makes more sense for it to just be in there. I will update and also add it to the `quire-starter-theme` should I add the `netlify.toml` file to? I guess that is why I didn't add the copy paste part is that I thought that might in there also.

Reply:
I will write it more like if you have to add commands more than the ones provided

Reply:
@naeluh sounds good, and I think it's okay for people to have to add the netlify.toml file themselves

Reply:
Ok I think that should work, I am going to create a pull request for starter theme